apps/vanilla/src/pages/bookCatalog/css/typography.css

```
@@ -0,0 +1,24 @@
+.typography__logo {
+	font-size: var(--font-size-xl);
+	line-height: 24px;
+	font-weight: 700;
+}
+
+.typography__subtitle {
+	font-size: var(--font-size-lg);
+	line-height: 24px;
+	font-weight: 600;
+	text-transform: uppercase;
+}
+
+.typography__body {
+	font-size: var(--font-size-xl);
+	line-height: 32px;
+	font-weight: 400;
+}
+
+.typography__body_small {
+	font-size: var(--font-size-md);
+	line-height: 24px;
+	font-weight: 500;
+}
```

`__` is the designation of the `BEM element`. But `_` (or `--`)  is the designation of the` BEM modificator`. https://ru.bem.info/methodology/quick-start/#%D0%BC%D0%BE%D0%B4%D0%B8%D1%84%D0%B8%D0%BA%D0%B0%D1%82%D0%BE%D1%80

```suggestion
.typography_logo {
	font-size: var(--font-size-xl);
	line-height: 24px;
	font-weight: 700;
}

.typography_subtitle {
	font-size: var(--font-size-lg);
	line-height: 24px;
	font-weight: 600;
	text-transform: uppercase;
}

.typography_body {
	font-size: var(--font-size-xl);
	line-height: 32px;
	font-weight: 400;
}

.typography_body-small {
	font-size: var(--font-size-md);
	line-height: 24px;
	font-weight: 500;
}
```